target/snitch_cluster/sw/apps/snax-gemm-engine/data/data.h

```diff
@@ -0,0 +1,19 @@
+#include <stdint.h>
+
+uint32_t m = 8;
+uint32_t n = 8;
+uint32_t k = 8;
+
+uint8_t A[] = {1,  2,  3,  4,  5,  6,  7,  8,  9,  10, 11, 12, 13, 14, 15, 16,
+               17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32,
+               33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48,
+               49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64};
+uint8_t B[] = {1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,
+               1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,
+               1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1};
+uint32_t C_golden[] = {36,  36,  36,  36,  36,  36,  36,  36,  100, 100, 100,
```

Comment:
Your data is too nice (?).
What happens if you use negative integers?
What happens if your accumulator overflows? Will it wrap around? Will it clamp?

Reply:
Hi @JosseVanDelm @rgantonio, currently, the GEMM input is unsigned integer. I will check what should I do to support signed integers. The accumulator can not overflow currently, the output data width is safe enough. I have tried several different input matrixes (unsigned 8-bit), and the GEMM still gives the right results.

Reply:
@xiaoling-yi are there any differences changes in hardware? There shouldn't be IIUC for add/mult.
But seems good to make sure they don't change :laughing: